File: Lógica de programação/JavaScript/farmacia.js
function calcularTotalCompra(itens) {
    let total = 0;
    for (let i = 0; i < itens.length; i++) {
        total += itens[i].quantidade * itens[i].preco;
    }
    return total;
}

const itensCompra = [];

for (let i = 0; i < 3; i++) {
    let nome = prompt(`Informe o nome do item ${i + 1}:`);
    let quantidade = parseInt(prompt(`Informe a quantidade do item ${i + 1}:`));
    let preco = parseFloat(prompt(`Informe o preço unitário do item ${i + 1}:`));

    itensCompra.push({ nome, quantidade, preco });
}

let calcularTotalCompra

File: 1 Periodo/Lógica de programação/Python/farmácia/farmacia.py
def calcular_total_compra(itens):
    total = 0
    for item in itens:
        total += item['quantidade'] * item['preco']
    return total


itens_compra = []

for i in range(3):
    nome = input(f"Informe o nome do item {i + 1}: ")
    quantidade = int(input(f"Informe a quantidade do item {i + 1}: "))
    preco = float(input(f"Informe o preço unitário do item {i + 1}: "))
    itens_compra.append(
        {'nome': nome, 'quantidade': quantidade, 'preco': preco})

total_compra = calcular_total_compra(itens_compra)

forma_pagamento = input("Informe a forma de pagamento (pix ou cartão): ")

if forma_pagamento == "pix":
    total_compra *= 0.90
elif forma_pagamento == "cartão":
    total_compra *= 1.05

inserir_cpf = input("Deseja inserir o CPF? (sim ou não): ")

if inserir_cpf == "sim":
    total_compra *= 0.90

print(f"Total a pagar: R$ {total_compra:.2f}")
